### DISCUSSION TITLE
Why preferring manual .tar.bz2 tarball than generated?

### DISCUSSION THREAD

**Discussion by sewnie** [p-gen/smenu#35] [Q&A] [answered]
> Do not use the sources (.zip/.tar.gz) automatically generated by github. Use the .tar.bz2 file instead if needed.

Why? The differences i could find via `diff` are:
```
Only in generated: .clang-format
Only in generated: SUMMARY
Common subdirectories: generated/build-aux and manual/build-aux
Common subdirectories: generated/examples and manual/examples
Only in generated: simple_menu.gif
Only in generated: smenu.gif
Common subdirectories: gen/tests and man/tests
```
Less file size it seems?

**Comment by p-gen** [comment, marked as answer]
The .tar.bz2 contains the compressed tests in the tests directory and is generated by `make distcheck` to make it ready for pakaging.
